db/migrate/20200201005117_add_visible_to_strategies.rb

```diff
@@ -1,5 +0,0 @@
-class AddVisibleToStrategies < ActiveRecord::Migration[5.2]
-  def change
-    add_column :strategies, :visible, :boolean, default: true
```

Comment:
Why is this being deleted?

Reply:
Let's not modify this file. I would remove changes to this file from your PR. We shouldn't be modifying previous migration files in any way. This may be responsible for the test migration error in CircleCI.

Reply:
So you have to remove this change entirely from this pull request. This is still causing the test migration error in CircleCI. There are two ways you can do this: 

1) You should do this by squashing all your commits into one. Doing a soft reset and remove this file and re-commit everything else.

2) Revert the specific commit that made this change